README.md

```diff
@@ -1,4 +1,6 @@
-# PicCap - Hyperion Sender App | Ambilight for LG WebOS TVs  
+# PicCap - Hyperion Sender App | Ambilight for LG WebOS TVs 
+
+Selection of NV12 settings added
```

Comment:
I'd propose to either remove this line from the README, or add a dedicated changelog section which lists this as the change for version 0.5.0.

So far the changes were documented [per release](https://github.com/TBSniller/piccap/releases/tag/0.4.3), so I'd be in favor of just removing it here

Reply:
Thanks for the hint, I have removed

Reply:
That was quick - looks good!